### DISCUSSION TITLE
Options in Github - forums and Wiki - how to add to the page!?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#35958] [Discussions]
hello dear friends 

Options in Github - forums and Wiki - how to add to the page!?

hello dear all, i have a projectpage - and i want to add several things to it
- a forum and 
- a wikii

how to do that!? Can i add a wikipage on this link - can i setup the wiki here 
Welcome to the myproject wiki!  https://github.com/my_projectpage/myproject/wiki
Wikis provide a place in your repository to lay out the roadmap of your project, show the current status, and document software better, together.

any idea how to arrange all that !?

**Comment by airtower-luna** [comment]
If by "forum" you mean the "discussions" feature like here: Go to the repository settings and scroll down to the "Features" section. Enable "Wikis" and "Discussions".
